xcell/mappers/mapper_DECaLS.py

```diff
@@ -58,21 +49,37 @@ def __init__(self, config):
         self.stars = None
         self.bmask = None
 
-    def _get_angmask(self, cat):
-        bmask = hp.read_map(self.config['binary_mask'],
-                            verbose=False)
-        nside = hp.npix2nside(len(bmask))
-        ipix = hp.ang2pix(nside, cat['RA'], cat['DEC'],
-                          lonlat=True)
-        return bmask[ipix] > 0.
+    def get_catalogs(self):
+        if len(self.cat_data) == 0:
```

Comment:
then make this
```
if self.cat_data == None:
    self.cat_data = []
```
Just to be consistent with the other mappers.

Reply:
resolved!